pkgs/applications/misc/mainsail/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, stdenvNoCC
+, fetchzip
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "mainsail";
+  version = "2.4.1";
+
+  src = fetchzip {
+    url = "https://github.com/mainsail-crew/mainsail/releases/download/v${version}/mainsail.zip";
```

Comment:
Why are we not building it from source via `buildNpmPackage`?

Reply:
It's not worth the effort